### ISSUE TITLE
Delay before notification pops up

### ISSUE THREAD

**Issue by svdg** [needim/noty#367]
Noty version: 3.10

Is it possible to delay the appearance of the notification? So, wait 2 seconds before it pops up? And how about cookies? Would it be possible to show it only once?

**Comment by needim** [comment]
Hi @svdg, currently there is no support for that behaviour. Out of scope. 

But you can use setTimeout and some cross browser localstorage/cookie libraries for that.
